src/components/ComputeModal.js

```diff
@@ -239,15 +239,15 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
         machineType: desiredRuntime.machineType || defaultGceMachineType,
         ...(desiredRuntime.diskSize ? {
           diskSize: desiredRuntime.diskSize
-        } : {
+        } : (shouldUpdatePersistentDisk ? { diskSize: desiredPersistentDisk.size } : {
```

Comment:
I had a difficult time following the logic around here 🤯  (not because of your PR). Stepped through the code on the console debugger and it looked correct 🤞🏼

Reply:
oh yea...took me a while to get this right...do you wanna demo how you stepped through the code in debugger during walkthrough today :) ...I'd be interested to know how you did that

Thanks for the review btw!